### ISSUE TITLE
About the loss curves.

### ISSUE THREAD

**Issue by striveAgain** [thu-ml/Causal-Forcing#42]
Hi! Sorry to trouble you again.

Could you please share the loss curves for all stages? I found the curves for ODE and CD in the closed issues, but Stage 1 and Stage 3 are currently missing.

By the way, I initialized the casual model directly with the bidirectional ckpt and run inference without any training. But the results are terribly bad, almost all noise. Is this normal?

**Comment by zhuhz22** [comment]
Stage 1:

<img width="1183" height="396" alt="Image" src="https://github.com/user-attachments/assets/28de8cf1-0b5c-4ab2-a2f7-58b6ffeead94" />

Stage 3:

<img width="1184" height="363" alt="Image" src="https://github.com/user-attachments/assets/6295c436-dcfc-479b-9c30-947f54468cd2" />

As for your observation ` I initialized the casual model directly with the bidirectional ckpt and run inference without any training. But the results are terribly bad`, this is quite normal, and is exactly why we need AR diffusion distillation.

**Comment by WJohnnyW** [comment]
can you show the curve of critic loss and grad of stage 3, I always find the generator becomes better at first, them occurs grid-like, stripe-like artifacts  gradually along the training steps (artifacts fist at the end of rollout, then the mid and end, finally full rollout, you can see  #41 for example).  After debugging, I think the problem may be the critic can't catch up the generator, I try to increase the learning rate of critic same with generator. the phenomenon alleviates but still occurs after longer training. Can you provide some insight for this problem? @zhuhz22

**Comment by zhuhz22** [comment]
<img width="596" height="200" alt="Image" src="https://github.com/user-attachments/assets/7d7bb9c7-4f5f-4a4b-ab0b-fffb1ff457a3" />
@WJohnnyW
